app.py

```diff
@@ -660,17 +660,30 @@ def prepare_3dsmax_launch(self, app_args):
         return app_args
 
 
-    def prepare_3dsmaxplus_launch(self, context, app_args):
+    def prepare_3dsmaxplus_launch(self, context, app_args, app_path):
         """
         3DSMax Plus specific pre-launch environment setup.
 
         Make sure launch args include a bootstrap to load the python engine:
         3dsmax.exe somefile.max -U PythonHost somescript.py
         """
         engine_path = tank.platform.get_engine_path("tk-3dsmaxplus", self.tank, context)
+
         if engine_path is None:
             raise TankError("Path to 3dsmaxplus engine (tk-3dsmaxplus) could not be found.")
 
+        # This is a fix for PySide problems in 2017+ versions of Max. Now that
+        # Max ships with a full install of PySide, we need to ensure that dlls
+        # for the native Max install are sourced. If we don't do this, we end
+        # up with dlls loaded from SG Desktop's bin and we have a mismatch that
+        # results in complete breakage.
+        paths = os.environ.get("PATH", "").split(";")
```

Comment:
can we use the [prepend path to env var](http://developer.shotgunsoftware.com/tk-core/utils.html#sgtk.util.prepend_path_to_env_var) method?